src/components/policy/policy_regular/test/policy_manager_impl_test.cc

```diff
@@ -1988,6 +1988,15 @@ TEST_F(PolicyManagerImplTest2,
   EXPECT_EQ(0u, ret.size());
 }
 
+TEST_F(PolicyManagerImplTest2, SetMetaInfo_SetCCPUVersion_SUCCES) {
+  const std::string ccpu_version = "ccpu_version";
+  const std::string wersCountryCode = "wersCountryCode";
+  const std::string language = "language";
+
+  manager->GetCache()->SetMetaInfo(ccpu_version, wersCountryCode, language);
+  EXPECT_EQ((manager->GetCache())->GetCCPUVersionFromPT(), ccpu_version);
```

Comment:
@ydementieiev Pattern is (EXPECTED_VAL, ACTUAL_VAL)

Reply:
@AByzhynar  [done](https://github.com/smartdevicelink/sdl_core/pull/3306/commits/88aa98a5d715e146c6f711fcb8219d8f6fde1873#diff-a431099559e14c2a2f7e1bcb7068ed0dL1997)